README.md

```diff
@@ -0,0 +1,46 @@
+[![Docker Image Version](https://img.shields.io/docker/v/sergii.ivanov/redis-commander?sort=date&label=Version)](https://hub.docker.com/r/sergii.ivanov/redis-commander/tags)
+[![Docker Image Size](https://img.shields.io/docker/image-size/sergii.ivanov/redis-commander?label=Image%20Size)](https://hub.docker.com/r/sergii.ivanov/redis-commander/tags)
+[![Docker Pulls](https://img.shields.io/docker/pulls/sergii.ivanov/redis-commander?label=Pulls)](https://hub.docker.com/r/sergii.ivanov/redis-commander)
+[![Docker Stars](https://img.shields.io/docker/stars/sergii.ivanov/redis-commander?label=Docker%20Stars)](https://hub.docker.com/r/sergii.ivanov/redis-commander)
+
+# Redis Commander
+
+Redis web management tool written in node.js
+
+# Image reason
+
+No special any reason. This just our infrastructure image.
+
+# Spec
+
+## Environment variables
+
+No any variables
```

Comment:
Pls add definition of `REDIS_COMMANDER_BG_COLOR`

Reply:
set the option color:
--env REDIS_COMMANDER_BG_COLOR=#......

this definition is correct?
maybe like this: set the value at the docker run